cpp/src/ice2slice/Gen.cpp

```diff
@@ -18,55 +19,53 @@ namespace
         TypePtr type;
         bool optional;
         int tag;
-        ParamDeclPtr param; // 0 == return value
+        ParamDeclPtr param; // nullptr == return value
```

Comment:
It would be good to refactor this ParamInfo in a follow-up PR.

ParamDecl contains all this information already (name, type etc.). I am sure we can do better.